### ISSUE TITLE
Exact costs used in darknetAB YoloV3

### ISSUE THREAD

**Issue by eyalzubia** [AlexeyAB/darknet#1283]
Hi, I noticed that the costs are slightly different than what's specified in the original YoloV3 paper. Here there's a weighted MSE for the coordinates (scaled by an inverse function of the bounding box size); the confidence appears to be a softmax with cross entropy, etc. 
Could @AlexeyAB  or anyone specify what costs (with activation functions and weights) are used in this fork? 

Thanks very much.
